test/acceptance/features/postgresqlOperatorExamples.feature

```diff
@@ -0,0 +1,331 @@
+@examples
```

Comment:
are we sure that `@examples` scenarios are passing? How about that we create additional job that can run examples on demand/manually? https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/ That should be a part of the current PR.